gpiozero/boards.py

```diff
@@ -1185,3 +1187,375 @@ def on(self):
     def off(self):
         self.value = False
 
+class SevenSegmentDisplay(LEDBoard):
+    """
+    Extends :class:`LEDBoard` for a 7 segment LED display 
+
+    7 segment displays have either 7 or 8 pins, 7 pins for the digit display 
+    and an optional 8th pin for a decimal point. 7 segment displays 
+    typically have either a common anode or common cathode pin, when
+    using a common anode display 'active_high' should be set to False.
+    Instances of this class can be used to display characters or control 
+    individual leds on the display. For example::
+
+        from gpiozero import SevenSegmentDisplay
+
+        seven = SevenSegmentDisplay(1,2,3,4,5,6,7,8,active_high=False)
+        seven.display("7")
+    
+    :param int \*pins:
+        Specify the GPIO pins that the 7 segment display is attached to.
+        Pins should be in the LED segment order A,B,C,D,E,F,G,decimal_point 
+        (the decimal_point is optional).
+
+    :param bool pwm:
+        If ``True``, construct :class:`PWMLED` instances for each pin. If
+        ``False`` (the default), construct regular :class:`LED` instances. This
+        parameter can only be specified as a keyword parameter.
+
+    :param bool active_high:
+        If ``True`` (the default), the :meth:`on` method will set all the
```

Comment:
Perhaps the documentation here should talk about common-anode and common-cathode, rather than talking about HIGH and LOW?